src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs

```diff
@@ -979,6 +1009,11 @@ public async Task ForceRenew()
                 if (Server.Certificate.Certificate == null)
                     throw new InvalidOperationException("Cannot force renew this Let's Encrypt server certificate. The server certificate is not loaded.");
 
+                if (LoggingSource.AuditLog.IsInfoEnabled)
+                {
+                    LogAuditFor("Certificates", "RENEW", "Renew server certificate");
```

Comment:
```suggestion
                    LogAuditFor("Certificates", "RENEW", "Server certificate");
```

Reply:
Any chance to also state the thumbprints - old and new? E.g. like ABCD375687165 -> ABCD375687166

Reply:
done